### DISCUSSION TITLE
研究 OpenAI 第一篇： API 介紹

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#914] [General]
OpenAI API 可以應用於幾乎任何涉及理解或生成自然語言或代碼的任務。提供了一系列適合不同任務且不同功能級別的模型， 以及微調自定義模型的能力。這些模型可用於從內容生成到語義搜索和分類的所有用途。

這篇網站是工程師為了分析OpenAI的ChatGPT而誕生的，但因為ChatGPT才剛釋出不久，所以他們並沒有辦法透過API或paper去了解，因此先了解背後，第一篇是公司API文件的介紹，有稍微描述模型與功能，與自然語言處理有非常大的關係。

https://hello-kirby.hashnode.dev/openai-api

**Comment by manipulateed** [comment]
這個api 能做到如此的技術，相比經歷很多很大量的資料訓練，才能成就他背後的模型，但他帶來方便性似乎降低人在自己思考的能力，而過度依賴這個模型

**Comment by ruthchen1020** [comment]
我覺得解說得很好，之前有接觸過一點所以大概了解，感謝分享。
